getpid.go

```diff
@@ -5,7 +5,10 @@ import (
 	"github.com/BurntSushi/xgbutil/xprop"
 )
 
-func i3Running() bool {
+type runHook func() bool
```

Comment:
Why the type declaration when the type is not exported?

You could just use `var IsRunningHook = func() bool { }`, no?

Reply:
reads cleaner, but if that is your preference more than happy to fix